### PR TITLE
Views 💯 

### PR DESCRIPTION
**Pull request by leehaney254**
- Implemented my views according to the wireframe
- Added some styling to my views
- Implemented the links to navigate my page
- Populated my database with data
- NB: to populate database run:
  - `rails db:seed`

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,22 @@
...
+  <p><%= @user.bio %></p>
+</section>
+ <div class="d-flex justify-content-end"><button type="button" class="btn btn-info"><%= link_to "See all Posts", user_posts_path(@user.id), { class: "user-link" } %></button></div>
+<section>
+  <% @posts.each do |post| %>
+      <div class="container-fluid border">
+        <h3><%= post.title %></h3>
+        <p><%= post.text %></p>
+        <p>Comments: <%= post.comments_counter.nil? ? 0 : post.comments_counter %></p>
+        <p>Likes: <%= post.likes_counter.nil? ? 0 : post.likes_counter %></p>
```
Comment (brytebee) [inline app/views/users/show.html.erb:19]: Kindly render the 3 most recent posts by the user as shown in the wireframe below.

image

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1 +1,30 @@
...
+      </div>
+    <% end %>
+    <div class="container-fluid border">
+      <% @comments = Post.find(post.id).comments %>
+        <% @comments.each do |comment| %> 
+          <div class="d-flex align-items-center">
+            <h3><%= comment.user.name %></h3>:
+            <h5 class="mx-2"><%= comment.text %></h5> 
+          </div>   
+        <% end %>
```
Comment (brytebee) [inline app/views/posts/index.html.erb]: Likewise, show the 5 most recent comments on the post here.

image

**Hint**

```
post.recent_comments.each |do|
  // comment info
end
```

The above block point to the method below defined in the Post model.

https://github.com/leehaney254/blog-app/blob/f1a7aeb10f6668913686fd6bc02f2efb66b0e17e/app/models/post.rb#L11-L13

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @leehaney254👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great DB design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/index.html.erb (hunk 2 above)
Comment (leehaney254) [reply app/views/posts/index.html.erb]: Thank you @brytebee  💪 . I have fixed it.

**Review by franclobo — APPROVED** [review]
Hi @leehaney254, 

# STATUS: APPROVED 😃 

leonardo-dicaprio-leo-dicaprio

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Git flow ✔️ 
- Well done using seed 👏🏽 
- Well done adding pagination 👏🏽 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**